config/jobs/containerd/cri/containerd-cri-presubmit-jobs.yaml

```diff
@@ -1,115 +1,135 @@
 presubmits:
   containerd/cri:
```

Comment:
Can we keep the build job? It is still better to have a separate job for build.

We can of course still build again for each node e2e test.